public/app/core/components/NestedFolderPicker/NestedFolderList.tsx

```diff
@@ -209,9 +209,8 @@ function Row({ index, style: virtualStyles, data }: RowProps) {
         )}
 
         <label className={styles.label} id={labelId}>
-          <Text as="span" truncate>
-            {item.title}
-          </Text>
+          {/* TODO: text is not truncated properly, it still overflows the container */}
```

Comment:
@eledobleefe friendly reminder to double check your merges so we aren't readding comments that don't apply anymore 🙏

Reply:
Hi @ashharrison90,
I wrongly linked it to the [Tooltip task](https://github.com/grafana/grafana/pull/68578) regarding the `truncate` prop. Do you want me to delete this comment in that PR? Or should I do it in a new one now?

Reply:
i fixed it [here](https://github.com/grafana/grafana/pull/72186), more just an fyi than anything 👍